www/on/npm/%package/index.html.spt

```diff
@@ -26,16 +27,43 @@ url = 'https://npmjs.com/package/' + package.name
 </a>
 {% endblock %}
 
+{% block scripts %}
+<script>
+    $(document).ready(function() {
+        $('button.send').on('click', Gratipay.packages.post);
+    });
+</script>
+{{ super() }}
+{% endblock %}
+
 {% block content %}
-{% if user.ANON %}
 <h2>{{ _( '{npm_package} has not been claimed on Gratipay.'
         , npm_package=('<a href="' + url + '">' + 'npm/' + package_name + '</a>')|safe
          ) }}</h2>
-<p>{{ _('Is this yours? You can claim it on Gratipay with a couple clicks:') }}</p>
-{% include "templates/sign-in-using.html" %}
+{% if user.ANON %}
+    <p>{{ _('Is this yours? You can claim it on Gratipay with a couple clicks:') }}</p>
+    {% include "templates/sign-in-using.html" %}
 
-<h2>{{ _("What is Gratipay?") }}</h2>
-<p>{{ _("Gratipay helps companies and others pay for open source.") }}
-<a href="/about/">{{ _("Learn more") }}</a></p>
+    <h2>{{ _('What is Gratipay?') }}</h2>
+    <p>{{ _('Gratipay helps companies and others pay for open source.') }}
+    <a href="/about/">{{ _("Learn more") }}</a></p>
+{% else %}
+    <p>{{ _( 'Is this yours? You can claim it on Gratipay using any email address {a}on file{_a} in the maintainers field in the npm registry.'
+           , a=('<a href="https://registry.npmjs.com/' + package.name + '">')|safe
+           , _a='</a>'|safe
+            ) }}
+    {% if nemails == 0 %}
+    <p class="sorry">{{ _("Sorry, we didn't find any email addresses on file.") }}</p>
+    {% else %}
+    <input type="hidden" name="package_id" value="{{ package.id }}">
+    <ul>
+        {% for i, email in enumerate(package.emails) %}
```

Comment:
Done in 455f06c67e56027839582c62ceeca8bab12b5351.

![screen shot 2017-02-09 at 8 53 14 am](https://cloud.githubusercontent.com/assets/134455/22786113/376803f4-eea5-11e6-9060-be1e78a84047.png)